cookbook/integration_langchain.ipynb

```diff
@@ -476,7 +476,15 @@
    "source": [
```

Comment:
<p>This change removed the scoring of the trace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/753/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>